app/models/Group.js

```diff
@@ -66,6 +67,11 @@ module.exports = function(Sequelize, DataTypes) {
       }
     },
 
+    hostFeePercent: {
+      type: DataTypes.FLOAT,
+      defaultValue: constants.HOST_FEE_PERCENT
```

Comment:
I wanted to make it an INT value 0  to 100. Issue is that we won't be able to charge any non-integer percentage like 2.5% then. So, made it a float. With float, we could represent 5% as `5` or `0.05`. Currently, I'm representing 5% as `5`, 2.5% as `2.5`, which is easier for humans to understand. Otherwise, we'll be manually adding `0.05` or `0.10` or `0.025` and it's easy to be off by a decimal.